cmd/wails/4_build.go

```diff
@@ -52,6 +74,25 @@ func init() {
 			return fmt.Errorf("Unable to find 'project.json'. Please check you are in a Wails project directory")
 		}
 
+		// Set cross-compile
+		projectOptions.Platform = runtime.GOOS
+		if len(platform) > 0 {
```

Comment:
I'm going to pimp `https://github.com/leaanthony/slicer` here as you can simply do `if !supportedPlatforms.Contains(plat)` :-)

Reply:
nice!  I was just keeping what was in the original branch.

Reply:
Yeah, I should have revisited the whole project for these scenarios and tidied it up a bit